README.md

```diff
@@ -1,11 +1,33 @@
+# callhome.sg
+
 This is very much in the MVP stage.
 
-# What does it do?
+## What does it do?
 
 - Call a user-specified number
 - Save user preferences to the DB
 
-# How to run
+## How to run
+
+### Setup
+
+- [TypeScript](https://www.npmjs.com/package/typescript)
+- node 12 (preferably installed with versioning tool such as [Node Version Manager](https://github.com/nvm-sh/nvm))
+- [Docker and docker-compose](https://docs.docker.com/get-docker/), if you want to use the docker method
+- A `backend/.env` file, taking reference from `backend/.env.sample`
+
+Backend runs on port `4000` while Frontend runs on port `3000` and can be run using the instructions below.
+
+### Server
+
+#### npm script
+
+```bash
+cd backend
+npm run start-local
```

Comment:
We have the `start:dev` command, was there another motivation behind this?

Reply:
`ERROR: Cannot find "/app/backend/config/config.json". Have you run "sequelize init"?`

I get this error when running `docker run -it call-home`.

Reply:
did not see `start:dev`. removed `start-local`.